src/content/1.7/development/architecture/migration-guide/testing/behat.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Behat testing
+menuTitle: Behat testing
+weight: 20
+---
+
+# Behat testing
+
+{{% notice tip %}}
+It is important that you get familiar with the `Behats` principles before reading further. You can find the official behat documentation [here](https://docs.behat.org/en/latest/guides.html).
```

Comment:
```suggestion
It is important that you get familiar with the `Behats` principles before reading further. You can find the official Behat documentation [here](https://docs.behat.org/en/latest/guides.html).
```

Reply:
I will make it lowercase letters everywhere.
![Screenshot from 2021-05-13 17-03-15](https://user-images.githubusercontent.com/31609858/118136845-3a435d80-b40d-11eb-8b9d-19c618116cf2.png)

Reply:
I will make it lowercase letters everywhere and maybe wrap in backslashes or make it bold to be more visible.
![Screenshot from 2021-05-13 17-03-15](https://user-images.githubusercontent.com/31609858/118136845-3a435d80-b40d-11eb-8b9d-19c618116cf2.png)